.circleci/config.yml

```diff
@@ -21,9 +21,10 @@ jobs:
             mkdir -p $UNDERLAY_WS/src && cp $OVERLAY_WS/src/grid_map/tools/ros2_dependencies.repos \
               $UNDERLAY_WS/ros2_dependencies.repos
             cd $UNDERLAY_WS && vcs import src < ros2_dependencies.repos
-            DEBIAN_FRONTEND=noninteractive  rosdep install -y --ignore-src --skip-keys "pybind11_vendor" --from-paths src
+            DEBIAN_FRONTEND=noninteractive  rosdep install -y --ignore-src \
+              --skip-keys "pybind11_vendor" --skip-keys "ompl" --skip-keys "slam_toolbox" --skip-keys "octomap" --from-paths src
             colcon build --symlink-install --parallel-workers 1 --packages-up-to \
-              pcl_ros
+              pcl_ros octomap_msgs nav2_msgs
```

Comment:
what's the dependency on nav2_msgs?

Reply:
`nav2_costmap_2d` deprends on `nav2_msgs`. CI always fails when building `nav2_costmap_2d`, but I don't know why...

Reply:
I guess the reason is a lack of resource.

Reply:
@SteveMacenski Could you make circleci timeout longer?

Reply:
```

c++: fatal error: Killed signal terminated program cc1plus
compilation terminated.
```

That's not a timeout, that's a lack of available memory on the container

Reply:
@SteveMacenski Thank you for configuring CircleCI.
By the way, is it better to create `foxy` branch because some components of `foxy` is not compatible with those of `rolling`?
Are you working on binary release for now? https://github.com/ANYbotics/grid_map/issues/268

Reply:
done - foxy-devel added

I'm not working on a binary release right now, I think its blocked by other dependencies unreleased if memory serves